specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```diff
@@ -0,0 +1,1443 @@
+      "post": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "x-ms-examples": {
+          "Check Deployment Name": {
+            "$ref": "./examples/checkInferenceEndpointDeploymentName.json"
+          }
+        },
+        "operationId": "MachineLearningInferenceEndpointDeployment_CheckNameAvailability",
+        "description": "Checks that the inference endpoint deployment name is valid and is not already in use.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationNameParameter"
+          },
+          {
+            "name": "properties",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointDeploymentCheckNameAvailabilityParameter"
+            },
+            "description": "The name of the endpoint deployment within the specified resource group."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Operation to check the endpoint deployment name availability was successful.",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "description": "Azure subscription identifier.",
+      "in": "path",
+      "type": "string",
+      "required": true
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "description": "Name of the resource group in which workspace is located.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "WorkspaceNameParameter": {
+      "name": "workspaceName",
+      "description": "Name of Azure Machine Learning workspace.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "APIVersionParameter": {
+      "in": "query",
+      "name": "api-version",
+      "type": "string",
+      "description": "Version of Azure Machine Learning resource provider API.",
+      "required": true
+    },
```

Comment:
Is it possible to just reference these parameters that is defined somewhere else? FOr instance in `machineLearningWorkspces.json`